### DISCUSSION TITLE
no-console: error not working in react project

### DISCUSSION THREAD

**Discussion by DevilsAutumn** [eslint/eslint#16724] [Q&A]
Hi,
I added the rule ```'no-console': 'error'``` in .eslintrc.js but it is not giving any error for console.log. As mentioned in [this comment](https://github.com/eslint/eslint/issues/14008#issuecomment-781099293), i also tried adding  ```'no-console': ['error', { allow: [''] }]``` to  override the configuration for ```no-console``` from the extended config but its still not working.

<details >
<summary>.eslintrc.js</summary>
<br>
<pre>
module.exports = {
  root: true,
  parserOptions: {
    ecmaVersion: 2021,
    sourceType: 'module',
    ecmaFeatures: {
      jsx: true,
    },
  },
  settings: {
    react: {
      version: 'detect',
    },
  },
  env: {
    jest: true,
    browser: true,
    amd: true,
    node: true,
    es6: true,
  },
  extends: [
    'eslint:recommended',
    'plugin:react/recommended',
    'plugin:prettier/recommended', // Make this the last element so prettier config overrides other formatting rules
  ],
  rules: {
    'no-console': ['error', { allow: [''] }],
    'no-unused-vars': ['error', { vars: 'all', args: 'after-used', ignoreRestSiblings: false }],
    'prettier/prettier': ['error', {}, { usePrettierrc: true }],
  },
}

</pre>
</details>

I also have prettier installed .

Eslint version ^8.30.0
Im running command ```eslint --fix frontend/src/**/*.{js,jsx,json}```

Thanks!

**Comment by mdjermanovic** [comment]
Can you first check if the command you are using actually lints the file where you have `console.log`? You can add `-f tap` to the command and check if the file appears in the output. (the [tap formatter](https://eslint.org/docs/latest/user-guide/formatters/#tap) outputs all linted files, including those that don't have lint errors).

**Reply by DevilsAutumn** [reply]
This is the output, looks like the file is not been actually linted.
![Screenshot 2022-12-29 231209](https://user-images.githubusercontent.com/83907321/209989806-fa5607f7-f93b-47b5-9a61-b1c2f844624e.jpg)

**Reply by DevilsAutumn** [reply]
okay so its working correctly after removing eslintConfig from package.json of react-app:
```
"eslintConfig": {
    "extends": [
      "react-app",
      "react-app/jest"
    ]
  },
```
I hope removing it won't create any other problems?

**Reply by sam3k** [reply]
Hard to tell @DevilsAutumn. I'm wondering if you the have Eslint configurations in two different places and one is overriding the other. If you would like to confirm; please provide a reproducible demo.
